src/backend/distributed/commands/table.c

```diff
@@ -1070,6 +1077,30 @@ AlterTableCommandTypeIsTrigger(AlterTableType alterTableType)
 }
 
 
+/*
+ * DropsForeignKey returns true if the given AlterTableStmt drops a foreign key
+ * for the given relationId. False otherwise.
+ */
+static bool
+DropsForeignKey(Oid relationId, AlterTableStmt *alterTableStatement)
```

Comment:
Instead of passing `relationId`, we could do the following to pass less args as possible:

```c
	LOCKMODE lockmode = AlterTableGetLockLevel(alterTableStatement->cmds);
	Oid leftRelationId = AlterTableLookupRelation(alterTableStatement, lockmode);
```

Reply:
should we call this function as `AlterTableDropsForeignKey` (or something similar) to make what drops the foreign key more clear ?